interactive_templates/templates/v2/analysis/measures.py

```diff
@@ -133,6 +133,7 @@ def parse_args():
     parser = argparse.ArgumentParser()
     parser.add_argument("--breakdowns", action="append", default=[], required=False)
     parser.add_argument("--input-dir", type=str, required=True)
+    parser.add_argument("--output-dir", type=str, required=True)
```

Comment:
This doesn't actually appear to be used? Starting at lines 187 the output is currently saved to args.input_dir